health-services/geopode-adapter/src/main/resources/start.sh

```
@@ -0,0 +1,11 @@
+#!/bin/sh
+
+if [[ -z "${JAVA_OPTS}" ]];then
```

Shellcheck (0.10.0)

[warning] In POSIX sh, [[ ]] is undefined.

(SC3010)